core/src/test/scala/dagr/core/execsystem/TaskManagerTest.scala

```diff
@@ -128,10 +128,19 @@ class TaskManagerTest extends UnitSpec with OptionValues with LazyLogging with B
     tryTaskNTimes(taskManager = taskManager, task = task, numTimes = 1, taskIsDoneFinally = true, failedAreCompletedFinally = true)
   }
 
+  it should "run an empty task" in {
```

Comment:
Might be nice to include a test that has a small pipeline of `realTask ==> Task.empty ==> anotherRealTask`.